### DISCUSSION TITLE
Add Album Title to the gallery view

### DISCUSSION THREAD

**Discussion by semiligneous** [alangrainger/immich-public-proxy#8] [Feature Requests]
As per title - this would be nice.

It's called "Description" when editing a Share Link - which can be different to the Album Title.

If Description (of the Share Link) is set, it should use that. If it is empty, it should use the Album Title (my opinion).

**Comment by alangrainger** [comment]
Great suggestion! Added in v1.3.1 :)

**Comment by semiligneous** [comment]
Awesome! Speedy work! I see it in the <title> tag now - that looks great, thank you.

Would be possible to have it rendered in the page body itself? I guess that opens up "what font, what size, what position" questions which I appreciate you want to minimise options/complexity.

But it would be nice for recipients of shares to see the title on the page, I think?
